ocaml/src/disk_safety.ml

```diff
@@ -109,7 +109,11 @@ let get_disk_safety alba_client namespaces dead_osds =
             (get_dead_namespace_osds ~namespace_id) >>= fun r ->
           Lwt.return (Some (namespace, r)))
         (function
-         | Error.Albamgr_exn (Error.Namespace_does_not_exist, _) -> Lwt.return_none
+         | Nsm_model.Err.Nsm_exn (Nsm_model.Err.Namespace_id_not_found, _)
+         | Error.Albamgr_exn (Error.Namespace_does_not_exist, _) ->
+            (* getting disk-safety might race with namespace deletion.
```

Comment:
Could you provide a log here so we can catch it somewhere and log it in our albacli wrapper.
Throwing away the info in this thread would seem a waste.
(for instance what namespace was not available)